### PR TITLE
Italian translation update.

### PR DESCRIPTION
**Pull request by Mek101**
I can't find a good translation of the "match" verb with the same meaning. Open to suggestions.

### CODE REVIEW

**Comment by Mek101** [comment]
Opening new pull request after fixing build error

**Comment by danieleds** [comment]
I'd say
"Trova solo parole intere"

In addition, try to not use the "Camel Case" style: that's used a lot in English, but not in Italian :)

**Comment by JuBan1** [comment]
Thanks for the updates. Since @danieleds can actually read this (and I can't :)) I'll leave this for him to merge once he has time.

**Comment by danieleds** [comment]
Looks very good

File: src/translations/notepadqq_it.ts [hunk 1]
```diff
@@ -1728,190 +1731,190 @@ Vuoi salvare comunque?</translation>
...
         <location filename="../ui/main.cpp" line="92"/>
         <source>Notepadqq will ask for root privileges whenever they are needed if either &apos;kdesu&apos; or &apos;gksu&apos; are installed. Running Notepadqq as root is not recommended. Use --allow-root if you really want to.</source>
-        <translation type="unfinished"></translation>
+        <translation>Notepadqq richiederà i privilegi root ogni qual volta sono necessari se &apos;kdesu&apos; o &apos;gksu&apos; è installato. Eseguire Notepadqq come root non è raccomandato. Usare --allow-root se si vuole veramente.</translation>
     </message>
     <message>
         <location filename="../ui/stats.cpp" line="126"/>
         <source>Do you want to help?</source>
-        <translation type="unfinished"></translation>
+        <translation>Vuoi aiuto?</translation>
```
Comment (danieleds) [inline src/translations/notepadqq_it.ts]: @Mek101 can you change this to "Vuoi darci una mano?" ?

**Pull request merged**